modules/acquia_cms_headless/src/Plugin/AcquiaCmsTour/AcquiaHeadlessForm.php

```diff
@@ -229,7 +230,6 @@ public function submitForm(array &$form, FormStateInterface $form_state) {
 
     // Proceed with form save and configuration settings actions.
     // Set and save the form values.
-    $this->config('acquia_cms_headless.settings')->set('starterkit_nextjs', $acms_starterkit_nextjs)->save();
```

Comment:
@chandan-singh7929 Any reason we are removing this here ? I think this is taken care in separate ticket which @apathak18 is working on.

Reply:
@vishalkhode1 - As discussed, I am keeping this changes and @apathak18 will implement logic to make checkbox readonly and remove code for uninstalling the module on unchecked.